### PR TITLE
New Illustrator cheat sheet

### PR DESCRIPTION
**Pull request by navneetzz**
**What does your Instant Answer do?**
My Instant answer provide shortcut keys used in adobe illustrator.

**What is the data source for your Instant Answer? (Provide a link if possible)**
I've myself created this cheat sheet.

**Why did you choose this data source?**
I myself created this cheat sheet

**Are there any other alternative (better) data sources?**
There might be better data source.But according to me what I did is enough.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Graphic designers will get most out it.

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
No, this not connected to DuckDuckHack Instant Answer.

**Are you having any problems? Do you need our help with anything?**
No I'm not having any problem.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Social media sites.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot 2015-12-22 04 21 50
screenshot 2015-12-22 04 21 50

---
https://duck.co/ia/view/illustrator_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@navneetzz Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata post coming soon

**Comment by navneetzz** [comment]
https://duck.co/ia/view/illustrator_cheat_sheet

**Comment by moollaza** [comment]
@duckduckgo/duckduckhack-mentors can someone please help @navneetzz ? He needs to remove the extra MySQL file from this PR :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=illustrator cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/illustrator_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/illustrator.json [hunk 1]
```diff
@@ -0,0 +1,662 @@
+{
+    "id": "illustrator_cheat_sheet",
+    "name": "illustrator",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/illustrator.json]: @navneetzz lets capitalise i in illustrator

File: share/goodie/cheat_sheets/json/illustrator.json [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+    "description": "Graphics Editor",
+    "metadata": {
+        "sourceName": "IllustratorCheatSheet"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/illustrator.json]: @navneetzz  sourceurl is missing
If there is no sourceurl,there is no need of sourcename

File: share/goodie/cheat_sheets/json/illustrator.json [hunk 3]
```diff
@@ -0,0 +1,662 @@
...
+            {
+                "key": "[Shift] [C]",
+                "val": "Convert Anchor Point"
+            },
+            {
+                "key": "T",
+                "val": "Type"
+            },
+            {
+                "key": "[Shift]   [T]",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/illustrator.json]: @navneetzz extra space between two keys.

File: share/goodie/cheat_sheets/json/illustrator.json [hunk 4]
```diff
@@ -0,0 +1,662 @@
+{
+    "id": "illustrator_cheat_sheet",
+    "name": "illustrator",
+    "description": "Graphics Editor",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/illustrator.json]: @navneetzz Let this be  a little more descriptive.
like "Keyboard shortcuts for Adobe Illustrator, graphics editor"

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=illustrator cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/illustrator_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@navneetzz I'm a little concerned about the length of this IA. Can we remove some stuff to shorten it up, so that it's only the most common/essential shortcuts?

**Comment by navneetzz** [comment]
I've already removed most of the part of the IA that I originally wrote.I'm checking on it right now if it's further possible to remove and will update soon.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=illustrator cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/illustrator_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=illustrator cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/illustrator_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/illustrator.json [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+                "val": "Show/Hide Attributes Panel"
+            },
+            {
+                "key": "F12",
+                "val": "Revert"
+            }
+        ],
+        "Swatches Panel": [
+            {
+                "key": "[Ctrl] [Click] [New] [Swatch] [button]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/illustrator.json]: I think we can join the `New Swatch Button` into 1 -- no need to split it up into 3

File: share/goodie/cheat_sheets/json/illustrator.json [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+                "val": "Replace with another"
+            }
+        ],
+        "Viewing Artwork": [
+            {
+                "key": "F",
+                "val": "Toggle between screen modes"
+            },
+            {
+                "key": "Spacebar",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/illustrator.json]: Let's use `[Space]` most Cheat Sheets use that phrasing

**Comment by moollaza** [comment]
@navneetzz I'm going to merge this because its OK. Please make a followup PR to address the few comments I've made above.

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

File: share/goodie/cheat_sheets/json/illustrator.json (hunk 6 above)
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/illustrator.json]: @moollaza  you have merged it do I need to correct it ?

**Comment by moollaza** [comment]
@navneetzz congrats! It's live: https://duckduckgo.com/?q=illustrator%20cheat%20sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by moollaza** [comment]
@navneetzz I believe in merging this PR your commit removing the MySQL was merged as well and so it is no longer in the repo.

Please make a new PR adding it back, and make sure your branches are clean in the future so we can avoid issues like this :+1:

**Pull request merged**